### PR TITLE
Add Event Handling example to README

### PR DESCRIPTION
**Pull request by FranzSkuffka**
Also fix capitalization in some headings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ The motivation for this is similar to ngReact's:
 - [Install](#install)
 - [Usage](#usage)
 - [Features](#features)
-	- [the vue-component directive](#the-vue-component-directive)
-	- [the createVueComponent factory](#the-createvuecomponent-factory)
+	- [The vue-component directive](#the-vue-component-directive)
+	- [The createVueComponent factory](#the-createvuecomponent-factory)
+	- [Event Handling](#event-handling)
```
Comment (nicolaspayot) [inline README.md]: I would not add this intel as a feature, at the same level of the `vue-component` directive and `createVueComponent` factory. It works thanks to `vprops-*`, so I think it'd better be right after this piece of code: 
```javascript
<vue-component vprops="ctrl.person" />
// equals to
<vue-component vprops-first-name="ctrl.person.firstName" vprops-last-name="ctrl.person.lastName" />
```
And a little bit shorter (the whole bunch of code I wrote in the issue is slightly opinionated). I'd put it this way:
___
Events can bubble up from Vue to AngularJS components by binding functions references as `vprops-*`
```javascript
<vue-component vprops-on-click="ctrl.handleClick"></vue-component>
```
```javascript
const VComponent = Vue.component('hello-component', {
  props: {
    onClick: Function
  },
  render (h) {
    return (
      <button onClick={this.onClick}></button>
    )
  }
})
```

**Comment by nicolaspayot** [comment]
@FranzSkuffka should I close this PR?

**Comment by FranzSkuffka** [comment]
I have little time recently but I still think this part of the docs is quite important. From a user's point of view I insist on the parts of the documentation being added. I can understand that from your perspective things might have to look different, but then it would be best if you added the changes yourself.

If you prefer me doing it I will also do it. I find this part of the documentation essential.

**Comment by nicolaspayot** [comment]
Sounds good. I've just pushed a PR. Feel free to give your review. Thanks for your work :-)

**Pull request closed without merging**